app/src/components/market/sections/market_verify/market_verify.tsx

```diff
@@ -100,6 +100,8 @@ const MarketVerifyWrapper: React.FC<Props> = (props: Props) => {
 
   if (!loading && errorMessage) return <GenericError>{errorMessage || 'Failed to fetch curation data'}</GenericError>
 
+  const verificationBtnDisabled =
+    loading || typeof selection !== 'number' || !ovmAddress || verificationState != 1 || (cpk && cpk.cpk.isSafeApp())
```

Comment:
Why can't we verify with the safe app?

Reply:
`requestVerification` does [not use the CPK](https://github.com/protofire/omen-exchange/blob/master/app/src/services/cpk.ts#L425) but rather the signer/provider directly. When running as a safe app we don't have direct access to the signer.